.github/workflows/test-and-deploy.yml

```diff
@@ -34,6 +34,13 @@ jobs:
       - name: Checkout sendgrid-java
         uses: actions/checkout@v2
 
+      - name: Create GitHub Release
```

Comment:
Need to move this to after the `mvn install` step and include the jar asset: https://github.com/sendgrid/sendgrid-java/blob/99028bae4fed2c4c1cc0915f8119327e86c5b41f/Makefile#L7

Same for the other Java repos.